form/src/components/Question.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import Result from "./Result";
+
+const Question = () => {
+  const [question, setQuestion] = useState(" ");
+
+  const handleCheckInput = (e) => {
+    setQuestion(e.target.value);
+  };
+  if (question.match(/\?$/) === null) {
```

Comment:
this not the ternary actually. And, why should I repeat same elements with both conditions is satisfied? They'll be rendered all the time whether condition is satisfied or not.